proposed/http-middleware/middleware-meta.md

```diff
@@ -123,7 +120,7 @@ Middleware taking this approach generally has the following commonalities:
   the request for processing.
 * The middleware is passed 2 arguments during invocation:
   1. An object that represents an HTTP request.
-  2. A delegate that receives the request to dispatch next middleware in the pipeline.
+  2. A delegate that receives the request to dispatch next middleware in the stack.
```

Comment:
Could you elaborate why? See my post here: https://groups.google.com/d/msg/php-fig/a-obTBtOWnE/MS-fFw2dBwAJ